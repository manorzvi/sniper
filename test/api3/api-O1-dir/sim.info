{'cmdline': ['../../run-sniper',
             '-n',
             '1',
             '-d',
             'api-O1-dir',
             '--',
             './api-O1'],
 'git_revision': '',
 'host': 'butler',
 'pin_version': 'unknown',
 'rusage': (19.883386,
            0.09242399999999999,
            38416,
            0,
            0,
            0,
            8028,
            0,
            0,
            0,
            144,
            0,
            0,
            0,
            48362,
            22),
 'snipercmd': ['bash',
               '-c',
               '/home/manorz/sniper-7.4/lib/sniper -c /home/manorz/sniper-7.4/config/base.cfg --general/total_cores=1 --general/output_dir=/home/manorz/sniper-7.4/test/api3/api-O1-dir --config=/home/manorz/sniper-7.4/config/nehalem.cfg --config=/home/manorz/sniper-7.4/config/gainestown.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --traceinput/stop_with_first_app=false -g --traceinput/enabled=true -g --traceinput/emulate_syscalls=true -g --traceinput/num_apps=1 -g --traceinput/trace_prefix=/tmp/tmpLN8RAU/run_benchmarks'],
 't_elapsed': 25.838965892791748,
 't_start': 1633352554.823147,
 'tracecmds': [['/home/manorz/sniper-7.4/record-trace',
                '-o',
                '/tmp/tmpLN8RAU/run_benchmarks',
                '--pa',
                '-e',
                '1',
                '-s',
                '0',
                '-r',
                '1',
                '--follow',
                '--routine-tracing',
                '--',
                './api-O1']],
 'user': 'root',
 'vmem': 220901376L}
