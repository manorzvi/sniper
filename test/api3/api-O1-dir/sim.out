                                     | Core 0    
  Instructions                       |   40121622
  Cycles                             |   13725068
  IPC                                |       2.92
  Time (ns)                          |    5159800
  Idle time (ns)                     |         87
  Idle time (%)                      |       0.0%
Branch predictor stats               |           
  num correct                        |   10022008
  num incorrect                      |       1517
  misprediction rate                 |      0.02%
  mpki                               |       0.04
TLB Summary                          |           
  I-TLB                              |           
    num accesses                     |      15341
    num misses                       |         58
    miss rate                        |      0.38%
    mpki                             |       0.00
  D-TLB                              |           
    num accesses                     |      40259
    num misses                       |        103
    miss rate                        |      0.26%
    mpki                             |       0.00
  L2 TLB                             |           
    num accesses                     |        161
    num misses                       |        130
    miss rate                        |     80.75%
    mpki                             |       0.00
Cache Summary                        |           
  Cache L1-I                         |           
    num cache accesses               |      15341
    num cache misses                 |        891
    miss rate                        |      5.81%
    mpki                             |       0.02
  Cache L1-D                         |           
    num cache accesses               |      40259
    num cache misses                 |       1797
    miss rate                        |      4.46%
    mpki                             |       0.04
  Cache L2                           |           
    num cache accesses               |       2726
    num cache misses                 |       2483
    miss rate                        |     91.09%
    mpki                             |       0.06
  Cache L3                           |           
    num cache accesses               |       2523
    num cache misses                 |       2480
    miss rate                        |     98.30%
    mpki                             |       0.06
DRAM summary                         |           
  num dram accesses                  |       2480
  average dram access latency (ns)   |      72.75
  average dram queueing delay        |      19.32
  average dram bandwidth utilization |      0.00%
Coherency Traffic                    |           
  num loads from dram                |       1116
  num loads from dram cache          |          0
  num loads from remote cache        |          0
